scripts/translate-excel-to-json.ts

```diff
@@ -17,10 +17,14 @@ function read_locales() {
   return _.keyBy(data, 'path');
 }
 
+function stringify(result: ResultMap, lang: 'fi' | 'sv' | 'en') {
+  return JSON.stringify(flatToDeep(result, lang), null, 2).replace(/(\r\n|\r)/g, '\n');
```

Comment:
This regex doesn't do anything. The problem wasn't with actual line breaks, but with the manual `\n` inserted into one of the strings.

So there's a section like
```
    "hankkeenAlueForm": {
      "header": "Alueet",
      "instructions": "Hankkeelle voi piirttää useita geometrioita. Geometria alueet vaikuttavat hankkeen liikennehaittaindekseihin.\nPiirtotyökalut löytyvät karttanäkymän vasemmasta alalaidasta. Kynätyökalulla voi piirtää monikulmaisia geometrioita klikkaamalla kohtaan, johon haluat kulman. Neliötyökalulla saat muodostettua neliömäisiä alueita klikkaamalla ensin kartalle ja liikuttamalla hiirtä haluamaasi suuntaan. Uudelleenklikkaus \"sulkee\" neliön piirron.\nGeometriaa voi muokata \"tarttumalla\" geometrian reunasta ja vetämällä. Myös neliömäistä geometriaa voi muokata reunasta tarttumalla.\nVoit poistaa geometrian valitsemalla ensin geometrian klikkaamalla sitä, jolloin vasemman alalaidan piirtotyökaluihin ilmestyy roskakori ikoni, jota klikkaamalla geometrian voi poistaa."
    },
```

After exporting, editing something else and then importing, this has changed to:
```
    "hankkeenAlueForm": {
      "header": "Alueet",
      "instructions": "Hankkeelle voi piirttää useita geometrioita. Geometria alueet vaikuttavat hankkeen liikennehaittaindekseihin.\r\nPiirtotyökalut löytyvät karttanäkymän vasemmasta alalaidasta. Kynätyökalulla voi piirtää monikulmaisia geometrioita klikkaamalla kohtaan, johon haluat kulman. Neliötyökalulla saat muodostettua neliömäisiä alueita klikkaamalla ensin kartalle ja liikuttamalla hiirtä haluamaasi suuntaan. Uudelleenklikkaus \"sulkee\" neliön piirron.\r\nGeometriaa voi muokata \"tarttumalla\" geometrian reunasta ja vetämällä. Myös neliömäistä geometriaa voi muokata reunasta tarttumalla.\r\nVoit poistaa geometrian valitsemalla ensin geometrian klikkaamalla sitä, jolloin vasemman alalaidan piirtotyökaluihin ilmestyy roskakori ikoni, jota klikkaamalla geometrian voi poistaa."
    },
```

So the regex needed to tackle this is:
```typescript
.replace(/(\\r\\n|\\r)/g, '\\n')
```

Reply:
Doing the replacing this late breaks the update detection, since the instructions rows are always different. And I get the `\r\n` in console output.
```
❯ yarn locales:import
yarn run v1.22.19
$ ts-node -P ./scripts/tsconfig.json ./scripts/translate-excel-to-json.ts
Updated key "hankeForm.hankkeenAlueForm.instructions" {
  path: 'hankeForm.hankkeenAlueForm.instructions',
  fi: 'Hankkeelle voi piirttää useita geometrioita. Geometria alueet vaikuttavat hankkeen liikennehaittaindekseihin.\r\n' +
    'Piirtotyökalut löytyvät karttanäkymän vasemmasta alalaidasta. Kynätyökalulla voi piirtää monikulmaisia geometrioita klikkaamalla kohtaan, johon haluat kulman. Neliötyökalulla saat muodostettua neliömäisiä alueita klikkaamalla ensin kartalle ja liikuttamalla hiirtä haluamaasi suuntaan. Uudelleenklikkaus "sulkee" neliön piirron.\r\n' +
    'Geometriaa voi muokata "tarttumalla" geometrian reunasta ja vetämällä. Myös neliömäistä geometriaa voi muokata reunasta tarttumalla.\r\n' +
    'Voit poistaa geometrian valitsemalla ensin geometrian klikkaamalla sitä, jolloin vasemman alalaidan piirtotyökaluihin ilmestyy roskakori ikoni, jota klikkaamalla geometrian voi poistaa.',
  sv: 'SV: Hankkeelle voi piirttää useita geometrioita. Geometria alueet vaikuttavat hankkeen liikennehaittaindekseihin.\r\n' +
    'Piirtotyökalut löytyvät karttanäkymän vasemmasta alalaidasta. Kynätyökalulla voi piirtää monikulmaisia geometrioita klikkaamalla kohtaan, johon haluat kulman. Neliötyökalulla saat muodostettua neliömäisiä alueita klikkaamalla ensin kartalle ja liikuttamalla hiirtä haluamaasi suuntaan. Uudelleenklikkaus "sulkee" neliön piirron.\r\n' +
    'Geometriaa voi muokata "tarttumalla" geometrian reunasta ja vetämällä. Myös neliömäistä geometriaa voi muokata reunasta tarttumalla.\r\n' +
    'Voit poistaa geometrian valitsemalla ensin geometrian klikkaamalla sitä, jolloin vasemman alalaidan piirtotyökaluihin ilmestyy roskakori ikoni, jota klikkaamalla geometrian voi poistaa.',
  en: 'EN: Hankkeelle voi piirttää useita geometrioita. Geometria alueet vaikuttavat hankkeen liikennehaittaindekseihin.\r\n' +
    'Piirtotyökalut löytyvät karttanäkymän vasemmasta alalaidasta. Kynätyökalulla voi piirtää monikulmaisia geometrioita klikkaamalla kohtaan, johon haluat kulman. Neliötyökalulla saat muodostettua neliömäisiä alueita klikkaamalla ensin kartalle ja liikuttamalla hiirtä haluamaasi suuntaan. Uudelleenklikkaus "sulkee" neliön piirron.\r\n' +
    'Geometriaa voi muokata "tarttumalla" geometrian reunasta ja vetämällä. Myös neliömäistä geometriaa voi muokata reunasta tarttumalla.\r\n' +
    'Voit poistaa geometrian valitsemalla ensin geometrian klikkaamalla sitä, jolloin vasemman alalaidan piirtotyökaluihin ilmestyy roskakori ikoni, jota klikkaamalla geometrian voi poistaa.'
}
Wrote 336 translation keys
✨  Done in 3.63s.
```

Reply:
Noita \n ei enää ole tuolla käännöstiedostoissa, niin ton regexin voisi poistaa. Sen jälkeen tämä olisi nähdäkseni valmis mergattavaksi.

Reply:
Poistettu.